src/Symfony/Component/Yaml/Command/LintCommand.php

```diff
@@ -11,12 +11,14 @@
 
 namespace Symfony\Component\Yaml\Command;
 
+use Symfony\Component\Console\CI\GithubAnnotationsReporter;
```

Comment:
this requires putting a `conflict` rule to ensure it cannot use older versions of `symfony/console` which don't have the class.